src/pages/spawn/spawnVolume/SpawnVolumeTable.tsx

```diff
@@ -22,79 +22,100 @@ export const SpawnVolumeTable: React.FC<SpawnVolumeTableProps> = ({
   volumes,
 }) => {
   const { search } = useLocation();
-  const volume = parseQueryString(search)?.volume;
   const dataSource: TableVolume[] = useMemo(() => {
     const volumesCopy = [...volumes];
     volumesCopy.sort(sortByHost);
     return volumes.map((v) => ({
       ...v,
+      renderExpandedContent: (row: LeafyGreenTableRow<TableVolume>) => (
+        <SpawnVolumeCard volume={row.original} />
+      ),
     }));
   }, [volumes]);
-  return (
-    <SpawnTable
-      expandedRowRender={(record: TableVolume) => (
-        <SpawnVolumeCard volume={record} />
-      )}
-      columns={columns}
-      dataSource={dataSource}
-      defaultExpandedRowKeys={[volume as string]}
-    />
+
+  const selectedVolume = parseQueryString(search)?.volume;
```

Comment:
Same comment as above about using `useQueryParam` instead of manually parsing the query string.